### ISSUE TITLE
error in as.ggdend when no set("branches_lty") specified

### ISSUE THREAD

**Issue by LSanselme** [talgalili/dendextend#12]
Hello,

Thanks for that package, and for the very useful `as.ggdend` function! 
I think there is a little bug in it, tough:
```R
ggd1  <- 1:50 %>%
  dist() %>% 
  hclust %>%
  as.dendrogram %>% 
  as.ggdend
```
returns `Error in FUN(X[[i]], ...) : subscript out of bounds`.

It works fine when adding `set("branches_lty", 1)`, as in the vignette.

```R
ggd1  <- 1:50 %>%
  dist() %>% 
  hclust %>%
  as.dendrogram %>% 
  set("branches_lty", 1) %>% 
  as.ggdend

ggplot(ggd1)
```
As the other features of your vignette (`branches_k_color`, `branches_lwd`, `labels_colors`, `labels_cex`) seem to be optional, I wonder why this is not the case for `branches_lty` ?

**Comment by talgalili** [comment]
Thank you for the report @LSanselme 
The "branches_lty" option is relatively new, and I did not know of this option.
I'm currently at JSM 2015 conference, but once I get back home I'll look into it (probably next week)

Cheers,
Tal

**Comment by talgalili** [comment]
This issue is now fixed for dendextend version 1.1.1.
I will release this version to CRAN (but first I'd like for the bioinformatics paper to get published so I could update the CITATION file).
In the meantime, you can install the package using devtools by following the instructions from here:
https://github.com/talgalili/dendextend#installation
 

Best,
Tal
